electricpy/visu.py

```diff
@@ -356,38 +372,62 @@ class PowerCircle:
         Sending End Reactive Power, default = None
     Ss: complex
         Sending End Total Complex Power, default = None
-    A: float 
-        Transmission System ABCD Parameters, A, default = None     
-    B: float 
-        Transmission System ABCD Parameters, B, default = None         
-    C: float 
-        Transmission System ABCD Parameters, C, default = None     
-    D: float 
-        Transmission System ABCD Parameters, D, default = None 
+    A: float
+        Transmission System ABCD Parameters, A, default = None
+    B: float
+        Transmission System ABCD Parameters, B, default = None
+    C: float
+        Transmission System ABCD Parameters, C, default = None
+    D: float
+        Transmission System ABCD Parameters, D, default = None
     """
 
-    def __init__(self, power_circle_type: str, power_factor: float = None, Vr: complex = None, Vs: complex = None,
-                Pr: float = None, Qr: float = None, Sr: complex = None,
-                Ps: float = None, Qs: float = None, Ss: complex = None,
-                A: complex = None, B: complex = None, C: complex = None,
-                D: complex = None) -> None:
+    def __init__(
+        self,
+        power_circle_type: str,
+        power_factor: float = None,
+        Vr: complex = None,
+        Vs: complex = None,
+        Pr: float = None,
+        Qr: float = None,
+        Sr: complex = None,
+        Ps: float = None,
+        Qs: float = None,
+        Ss: complex = None,
+        A: complex = None,
+        B: complex = None,
+        C: complex = None,
+        D: complex = None,
+    ) -> None:
         r"""Initialize the class."""
         if C is not None:
-            assert abs(A*D - B*C - 1) < 1e-6, "ABCD Matrix is not a valid ABCD Matrix"
+            assert (
+                abs(A * D - B * C - 1) < 1e-6
+            ), "ABCD Matrix is not a valid ABCD Matrix"
 
         if power_circle_type.lower() == "receiving":
 
             if A != None and B != None and Vr != None:
-                self.radius, self.center, self.operating_point = PowerCircle._build_circle(A, B, "receiving_end", Vr, 
-                Pr, Qr, Sr, power_factor, Vs)
+                (
+                    self.radius,
+                    self.center,
+                    self.operating_point,
+                ) = PowerCircle._build_circle(
+                    A, B, "receiving_end", Vr, Pr, Qr, Sr, power_factor, Vs
+                )
             else:
                 raise ValueError("Not enough attributes to build circle")
 
         elif power_circle_type.lower() == "sending":
-            
+
             if B != None and D != None and Vs != None:
-                self.radius, self.center, self.operating_point = PowerCircle._build_circle(D, B, "sending_end", Vs, 
-                Ps, Qs, Ss, power_factor, Vr)
+                (
+                    self.radius,
```

Comment:
- [ ] This is clunky, there must be another way to clean it up so that it's better to understand visually.

Reply:
Can we do:

```python
self.radius, self.center, self.operating_point = PowerCircle._build_circle(
...
```

?